lib/seahorn/BvOpSem2ExtraWideMemMgr.hh

```diff
@@ -3,7 +3,7 @@
 #include "BvOpSem2Context.hh"
 #include "BvOpSem2RawMemMgr.hh"
 #include "BvOpSem2TrackingRawMemMgr.hh"
-#include "BvOpSem2WideMemManagerMixin.hh"
+#include "BvOpSem2WideMemManagerMixin.cc"
```

Comment:
looks like a bug. You should never include a `cc` file in a header file!

Reply:
fixed